content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -324,7 +324,7 @@ The application is responsible for reloading the token when it rotates. Periodic
 
 ## Service Account Issuer Discovery
 
-{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
```

Comment:
Did you mean:
```suggestion
{{< feature-state for_k8s_version="v1.20" state="beta" >}}
```
?

Reply:
Nice catch! Cherry picked the original and forgot that.

Reply:
Done